ray-operator/controllers/ray/common/pod_test.go

```diff
@@ -455,6 +457,33 @@ func TestHeadPodTemplate_WithServiceAccount(t *testing.T) {
 	}
 }
 
+func TestValidateHeadRayStartParams_OK(t *testing.T) {
+	input := instance.Spec.HeadGroupSpec.DeepCopy()
+	isValid, err := ValidateHeadRayStartParams(*input)
+	assert.Equal(t, true, isValid)
+	assert.Nil(t, err)
+}
+
+func TestValidateHeadRayStartParams_ValidWithObjectStoreMemoryError(t *testing.T) {
+	input := instance.Spec.HeadGroupSpec.DeepCopy()
+	input.RayStartParams["object-store-memory"] = "2000000000"
```

Comment:
minor: use the constant you defined as well?

Reply:
done in https://github.com/ray-project/kuberay/pull/332/commits/635ca71a36ad4da67b24a84163ebca523e8a05f6